src/main/java/com/suse/saltstack/netapi/parser/JsonParser.java

```diff
@@ -81,4 +89,29 @@ public Date deserialize(JsonElement jsonElement, Type type,
             }
         }
     }
+
+    private class StatsDeserializer implements JsonDeserializer<Stats> {
+
+        @Override
+        public Stats deserialize(JsonElement jsonElement, Type type,
+                JsonDeserializationContext jsonDeserializationContext)
+                throws JsonParseException {
+            try {
+                JsonObject stats = jsonElement.getAsJsonObject();
+                Applications app = gson.fromJson(stats.get("CherryPy Applications"), Applications.class);
+                HttpServer server = null;
+                for(Map.Entry<String, JsonElement> entry : stats.entrySet()){
+                    String key = entry.getKey();
+                    if(key.startsWith("CherryPy HTTPServer ")){
```

Comment:
I actually had them as `static final` at first but when i realized that i just used them once i thought inlining them would be better. Is there still an advantage of making them `static final` even if they are only used once?